src/Umbraco.Web.UI.Client/src/views/documenttypes/edit.controller.js

```diff
@@ -60,6 +62,8 @@
                 documentTypeId = $scope.model.id;
                 create = $scope.model.create;
                 noTemplate = $scope.model.notemplate;
+                isElement = $scope.model.iselement;
+                allowVaryByCulture = $scope.model.allowvarybyculture;
```

Comment:
@marcemarc thanks 😄 

I may be a bit picky, but could you change `$scope.model.iselement` and $scope.model.allowvarybyculture` to `$scope.model.isElement` and $scope.model.allowVaryByCulture` respectively?

Ideally these should be camel cased, as this follows the convention used for the parameters for the other overlays. I know `$routeParams.notemplate` is all lowercase, but IMO that is also wrong (we can perhaps fix that later).